### ISSUE TITLE
Play-UI using old language switcher pattern

### ISSUE THREAD

**Issue by adamliptrot-oc** [hmrc/accessibility#57]
Services pulling in play-language currently get the old pattern
https://github.com/hmrc/play-language/blob/b5a259d6d68c262b70ac7534899563a7648ba1dc/src/main/scala/views/language_selection.scala.html

This fails WCAG on several criteria
WCAG 3.1.2 Language of Parts
WCAG 2.4.4 Link Purpose (In Context)
WCAG 1.3.1 Info and Relationships

Plat-UI are looking at pulling play-language into the main play-frontend-hmrc and are investigating the possibility of doing the same for play-ui (incorporating the fixes in the process)
https://jira.tools.tax.service.gov.uk/browse/PLATUI-909

**Comment by shabana-ali** [comment]
Fixed
